Regression/WarpX-GPU-tests.ini

```diff
@@ -0,0 +1,655 @@
+
+[pml_x_ckc]
+buildDir = .
+inputFile = Examples/Tests/PML/inputs_2d
+runtime_params = warpx.do_dynamic_scheduling=0 algo.maxwell_fdtd_solver=ckc
+dim = 2
+addToCompileString = USE_GPU=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 2
+compileTest = 0
+doVis = 0
+analysisRoutine = Examples/Tests/PML/analysis_pml_ckc.py
+
+#[pml_x_psatd]
+#buildDir = .
+#inputFile = Examples/Tests/PML/inputs_2d
+#runtime_params = warpx.do_dynamic_scheduling=0
+#dim = 2
+#addToCompileString = USE_PSATD=TRUE USE_GPU=TRUE
+#restartTest = 0
+#useMPI = 1
+#numprocs = 2
+#useOMP = 0
+#numthreads = 2
+#compileTest = 0
+#doVis = 0
+#analysisRoutine = Examples/Tests/PML/analysis_pml_psatd.py
+
+[RigidInjection_lab]
+buildDir = .
+inputFile = Examples/Modules/RigidInjection/inputs_2d_LabFrame
+dim = 2
+addToCompileString = USE_GPU=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 2
+compileTest = 0
+tolerance = 1e-12
+doVis = 0
+compareParticles = 0
+analysisRoutine = Examples/Modules/RigidInjection/analysis_rigid_injection_LabFrame.py
+
+[RigidInjection_boost_backtransformed]
+buildDir = .
+inputFile = Examples/Modules/RigidInjection/inputs_2d_BoostedFrame
+dim = 2
+addToCompileString = USE_GPU=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 2
+compileTest = 0
+doVis = 0
+compareParticles = 0
+doComparison = 0
+aux1File = Tools/read_raw_data.py
+analysisRoutine = Examples/Modules/RigidInjection/analysis_rigid_injection_BoostedFrame.py
+
+[nci_corrector]
+buildDir = .
+inputFile = Examples/Modules/nci_corrector/inputs_2d
+runtime_params = amr.max_level=0 particles.use_fdtd_nci_corr=1
+dim = 2
+addToCompileString = USE_GPU=TRUE
+restartTest = 0
+useMPI = 1
+numprocs = 2
+useOMP = 0
+numthreads = 2
+compileTest = 0
+doVis = 0
+doComparison = 0
+analysisRoutine = Examples/Modules/nci_corrector/analysis_ncicorr.py
+
+# [nci_correctorMR]
+# buildDir = .
+# inputFile = Examples/Modules/nci_corrector/inputs_2d
+# runtime_params = amr.max_level=1 particles.use_fdtd_nci_corr=1
+# dim = 2
+# addToCompileString = USE_GPU=TRUE
+# restartTest = 0
+# useMPI = 1
+# numprocs = 2
+# useOMP = 0
+# numthreads = 2
+# compileTest = 0
+# doVis = 0
+# doComparison = 0
+# analysisRoutine = Examples/Modules/nci_corrector/analysis_ncicorr.py
+
+# [ionization_lab]
+# buildDir = .
+# inputFile = Examples/Modules/ionization/inputs_2d_rt
+# dim = 2
+# addToCompileString = USE_GPU=TRUE
+# restartTest = 0
+# useMPI = 1
+# numprocs = 2
+# useOMP = 0
+# numthreads = 1
+# compileTest = 0
+# doVis = 0
+# analysisRoutine = Examples/Modules/ionization/analysis_ionization.py
+
+# [ionization_boost]
+# buildDir = .
+# inputFile = Examples/Modules/ionization/inputs_2d_bf_rt
+# dim = 2
+# addToCompileString = USE_GPU=TRUE
+# restartTest = 0
+# useMPI = 1
+# numprocs = 2
+# useOMP = 0
+# numthreads = 1
+# compileTest = 0
+# doVis = 0
+# analysisRoutine = Examples/Modules/ionization/analysis_ionization.py
```

Comment:
Do we plan to re-activate these tests?

Reply:
Yes, I believe there are open Issues about fixing these on GPUs. We disabled them so that the tests won't fail all the time until someone gets to that.